bson/_cbsonmodule.c

```diff
@@ -3051,6 +3083,21 @@ static int _cbson_traverse(PyObject *m, visitproc visit, void *arg) {
     Py_VISIT(GETSTATE(m)->MaxKey);
     Py_VISIT(GETSTATE(m)->UTC);
     Py_VISIT(GETSTATE(m)->REType);
+    Py_VISIT(GETSTATE(m)->_type_marker_str);
```

Comment:
_type_marker_str was missed here in the initial PR. Could you open a bug ticket to fix that in 4.4.1? We'll need to create a v4.4 branch to backport this change.